### ISSUE TITLE
Strategy for handling legacy function duplication

### ISSUE THREAD

**Issue by SOLV-Code** [avelez-espino/forecastR_phase4#19]
## The Challenge
a lot of the model-specific legacy functions seem to be identical or almost identical. 

## Proposed Approach
Have the full function in 1 place, and in the other place just give that original function a new name

## Example
* The folder *n1* has a script *Review Code - Naive One_functions.r*
* In that script, there is a function *datalist.naiveone* 
* Treat this as the master version
* The folder *n3* has a script *Review Code - Average Three_functions.r* which includes a function *datalist.avgthree*
* **If they are identical**: replace the full code duplicate code chunk to create this function with 
```
datalistavgthree <- datalist.naiveone
```
* **If they are different**: create a new list object that includes both variations, and call them from there. This way they are still in 1 place, at least, and it sets the stage for future streamlining.
```
datalistnaiveone<- datalist.master$naiveone
```
and
```
datalistavgthree <- datalist.master$avgthree
```

**Comment by MichaelFolkes** [comment]
under the condition of "different", I'm not sure it makes sense to put them in a list as this won't be applicable in the package.

**Comment by SOLV-Code** [comment]
good point. Maybe then just put them all together into  their own .R file, so that they are at least in 1 place, and can be compared easily?

**Comment by SOLV-Code** [comment]
In R File Common Functions

datalist.general <- function

source a separate startup file with
datalist.avgthree <- datalist.general
datalist.naiveone <- datalist.general

**Comment by MichaelFolkes** [comment]
I agree
